### PR TITLE
Remove link ganache -> ee

### PR DESCRIPTION
**Pull request by jtakalai**
when developing EE we might want ganache without EE

also it doesn't really need it, does it? Rather the other way around.

I guess EE should actually start ganache? Maybe add the link there?

### CODE REVIEW

**Comment by jtakalai** [comment]
Ha, now I remember why there should be such link: streamr-ganache wants to run the "test products" into the marketplace, and those would come from the mysql database through EE.

HOWEVER, it would only work if EE were completely started first, so in addition to the "link", ganache container would have to wait until EE is up and can give the products. Or maybe it should try to get them directly from mysql? Then they could just be in the mysql, though I'm afraid they're added through migrations... but I guess mysql isn't completely empty, it already has some migrated state?

**Comment by kare** [comment]
LGTM👍

File: docker-compose.override.yml [hunk 1]
```diff
@@ -39,6 +39,20 @@ services:
...
           links:
               - data-api
+    ganache:
+        container_name: streamr_dev_ganache
+        image: streamr/streamr-ganache:dev
+        ports:
+          - "8545:8545"
+        restart: on-failure
+        environment:
+            EE_URL: http://engine-and-editor:8081/streamr-core
```
Comment (hpihkala) [inline docker-compose.override.yml]: The URL given here won't work without the `links` section which was removed. This line leads to believe that the ganache container accesses `engine-and-editor` for some reason?

Comment (jtakalai) [reply docker-compose.override.yml]: it does, for getting test products that are added to Marketplace contract

**Comment by jtakalai** [comment]
outdated by now

**Pull request closed without merging**